src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,89 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT = 1 << 4;
+    private static final float LOAD_FACTOR = 0.75f;
+    private LinkedList<Entry<K,V>>[] buckets;
+    private V null1;
+    private int size;
+
+    public MyHashMap() {
+        buckets = new LinkedList[DEFAULT];
+        for (int i = 0; i < DEFAULT; i++) {
+            buckets[i] = new LinkedList<>();
+        }
+    }
+
+    private static class Entry<K, V> {
```

Comment:
Move the private inner class 'Entry' to be located after the public methods in the 'MyHashMap' class to maintain a logical organization of the class members.